jedi/settings.py

```diff
@@ -83,10 +85,17 @@
 """
 The path where the cache is stored.
 
-On Linux, this defaults to ``~/.cache/jedi/``, on OS X to
-``~/Library/Caches/Jedi/`` and on Windows to ``%LOCALAPPDATA%\\Jedi\\Jedi\\``.
+The cache-directory can be explicitly set by an environment-variable
+``JEDI_CACHE_DIR``.
+
+If this env-var is not set, then
```

Comment:
Can you revert this sentence? I don't like the grammar.

Reply:
reverted.